src/sass/patterns/_data-tables.scss

```diff
@@ -0,0 +1,12 @@
+.slds-table_header-hidden {
+  thead {
+    clip: rect(0 0 0 0);
+    height: 1px;
+    overflow: hidden;
+    position: absolute;
+    margin: -1px;
+    text-transform: none;
+    white-space: nowrap;
+    width: 1px;
+  }
```

Comment:
@stacyk If you prefer, we could just import and use their `assistive-text` mixin directly? Or maybe we want to retain control, in case their mixin changes...

Reply:
I didn't use all of their properties, and I didn't add all the !important, but we could try it. You mentioned it affected layout so I thought Id pick and choose what I needed.